.github/workflows/pr.yaml

```diff
@@ -1,9 +1,9 @@
-name: GitHub CI
+name: PR commit checks and PR target branch commit checks
```

Comment:
I don't think we want to remove the name field. I agree to make it shorter.

We can use comments in the yaml file to clarify intent; they will not appear in the UI but they will be visible in the source tree

I don't think we need to identify with "GH"; after all the directory these live in is .github/workflows; it is quite obvious that any file in there is a GitHub workflow.

Reply:
> I don't think we need to identify with "GH"; after all the directory these live in is .github/workflows; it is quite obvious that any file in there is a GitHub workflow.

We want GH (or equivalent) because these names are used as entries in an index that includes both GitHub and non-GitHub checks:

* PR commit / build-tests(ubuntu-22.04,clang,minimal) (pull_request)  Successful in 7m
* PR commit / build-tests(ubuntu-22.04,clang,maximus) (pull_request)  Successful in 15m
* Jenkins(build test)  Pending

In the above sample PR page output, GitHub and Jenkins both test "PR commit"[^1]. it would be good to add "GH" (or equivalent) just like Jenkins adds "Jenkins". Doing so will improve naming consistency and may reduce the number of folks that do not know where non-Jenkins checks are coming from.

[^1]: Well, they actually do not test "PR commit" -- they test [commit Y not X](https://github.com/squid-cache/squid/pull/1917#discussion_r1813340507)) -- but this part of the discussion is about "GH" prefix, not a misleading (especially for staged tests!) "PR commit" description that follows that prefix.